src/core/border_router/routing_manager.cpp

```diff
@@ -817,6 +817,45 @@ bool RoutingManager::NetworkDataContainsUlaRoute(void) const
     return contains;
 }
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_REACHABLITY_CHECK_ICMP6_ERROR_ENABLE
+
+void RoutingManager::CheckReachbilityToSendIcmpError(const Message &aMessage, const Ip6::Header &aIp6Header)
+{
+    Ip6::MessageInfo messageInfo;
+
+    VerifyOrExit(IsRunning() && IsInitalPolicyEvaluationDone());
```

Comment:
```suggestion
    VerifyOrExit(IsRunning() && IsInitialPolicyEvaluationDone());
```

Reply:
Thanks for noticing this :).

Since this typo is already used in multiple places in the code, I'd suggest and prefer to leave it unchanged in this PR and address it separately in a dedicated, smaller PR later.

Reply:
Fixed it in new push.